src/Pos/Explorer/Txp/Local.hs

```diff
@@ -130,9 +136,9 @@ eTxProcessTransaction itw@(txId, TxAux {taTx = UnsafeTx {..}}) = do
                      flip evalUtxoStateT resolved $
                      flip runReaderT eet $
                      runExplorerReaderWrapper $
+                     flip runReaderT genStks $
```

Comment:
Eeh, i'd prefer a specific reader environment with one datatype instead of two `ReaderT` layers.